PathToRegex.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = "PathToRegex"
+  s.version      = "0.2.2"
+  s.summary      = "A Swift library translating paths with wildcards into regular expressions"
+  s.homepage     = "https://github.com/crossroadlabs/PathToRegex"
+  s.license      = { :type => 'Apache 2.0', :file => 'LICENSE' }
+  s.author       = { "Crossroad Labs" => "daniel@crossroadlabs.xyz" }
+  s.source       = { :git => "https://github.com/crossroadlabs/PathToRegex.git", :tag => "#{s.version}" }
+  s.source_files = 'PathToRegex/**/*'
+
+  s.ios.deployment_target = '8.0'
```

Comment:
should I just copy whatever is already here
https://github.com/crossroadlabs/Regex/blob/master/CrossroadRegex.podspec#L11-L14